application/cms/views.py

```diff
@@ -972,16 +974,19 @@ def process_input_data():
 @login_required
 def process_auto_data():
     """
-    This is an AJAX endpoint for the AutoDataGenerator data standardiser
+    This is an AJAX endpoint for the EthnicityClassificationFinder data standardiser
 
     It is called whenever data needs to be cleaned up for use in second generation front end data tools
     (chartbuilder 2 & potentially tablebuilder 2)
 
-    :return: A list of processed versions of input data using different "presets"
+    :return: A list of processed versions of input data using different "classifications"
     """
-    if current_app.auto_data_generator:
+    if current_app.classification_finder:
         request_json = request.json
-        return_data = current_app.auto_data_generator.build_auto_data(request_json["data"])
+
+        valid_classifications_data = current_app.classification_finder.find_classifications(request_json["data"])
+        return_data = Builder2FrontendConverter(valid_classifications_data).convert_to_builder2_format()
+
         return json.dumps({"presets": return_data}), 200
     else:
         return json.dumps(request.json), 200
```

Comment:
Not really code you've touched here, but it feels like this should probably be a 500 if we don't have a classifier set up?

Reply:
it always returns the "Custom" classifier so it isn't an issue - I'll see what the deal is with that else clause